android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -202,20 +203,9 @@ private boolean shouldIgnoreNotification(Bundle bundle){
         }
     }
 
-    private boolean isAppOnForeground(Context context) {
-        boolean appIsInForeground = false;
-        ActivityManager activityManager = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
-        List<ActivityManager.RunningAppProcessInfo> appProcesses = activityManager.getRunningAppProcesses();
-        if (appProcesses == null) {
-            appIsInForeground = false;
-        }
-        final String packageName = context.getPackageName();
-        for (ActivityManager.RunningAppProcessInfo appProcess : appProcesses) {
-            if (appProcess.importance == ActivityManager.RunningAppProcessInfo.IMPORTANCE_FOREGROUND && appProcess.processName.equals(packageName)) {
-                appIsInForeground = true;
-            }
-        }
-        return appIsInForeground;
+    private boolean isAppInForeground(Context context) {
```

Comment:
This approach will definitely work but I am not too keen on this as it is adding more complexity and more code in 3 places: (RN PN, RN PNM, and RB). I think we should use the RN app state to clear the group in view when app goes in background and set it again when the user comes back to group screen.

Reply:
@peresbri I am using the AppState in rivalbet-mobile

Reply:
https://github.com/apthletic/rivalbet-mobile/blob/31fdfdc908d7f9cd2b4e3e700314007e86dcfddc/src/App.tsx#L186

Reply:
Approving so we can test and try the other method later as agreed.